pandas/tests/indexes/conftest.py

```diff
@@ -8,6 +8,7 @@
 import pandas.util.testing as tm
 
 
+# add inteval index?
```

Comment:
yes (you *might* want to do this as a precursor PR if other tings need adjustment)